tests/models/test_models_vae.py

```diff
@@ -312,10 +312,32 @@ def test_stable_diffusion(self):
         assert sample.shape == image.shape
 
         output_slice = sample[-1, -2:, -2:, :2].flatten().float().cpu()
-        expected_output_slice = torch.tensor([0.9858, 0.9262, 0.8629, 1.0974, -0.091, -0.2485, 0.0936, 0.0604])
+        expected_output_slice = torch.tensor([0.0093, 0.6385, -0.1274, 0.1631, -0.1762, 0.5232, -0.3108, -0.0382])
```

Comment:
Sorry, I don't follow. Why are these values being changed? The decoded output shouldn't change no? What am I missing out on?

Reply:
Indeed - the decoder behavior already worked fine (and this PR doesn't change it). However, this test doesn't just run the decoder; this test does a full round-trip through `AutoencoderTiny.forward` (which includes the encoder, as well as quantization / dequantization of latents).

The existing round trip code had two bugs (fixed in this PR):
1. No  [-1, 1] -> [0, 1] rescaling of the inputs in EncoderTiny
2. No [0, 255] -> [0, 1] rescaling in AutoencoderTiny.forward

It looks like the golden values in this test were computed using the previous buggy round-trip code, so because I've fixed the bugs in the round-trip code, I also need to change the golden values in this test.

Reply:
Thanks for explaining. Now it all makes sense to me! I will get the golden values for you from the internal CI runner to ensure exact matches.

Reply:
Ah it's passing already, great then!